jest.config.ts

```diff
@@ -5,6 +5,12 @@
 
 import type {Config} from 'jest';
 
+import nextJest from 'next/jest.js';
```

Comment:
There's something really funky with Nextjs/Jest together here that I really don't understand.

I found this solution from here (most recent reply): https://github.com/vercel/next.js/issues/40183
Here's another related issue in their Github: https://github.com/vercel/next.js/issues/58595

```
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    C:\Users\Eric\WebstormProjects\osrs-dps-calc-wiki\node_modules\d3-array\src\index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){export {default as bisect, bisectRight, bisectLeft, bisectCenter} from "./bisect.js";
                                                                                      ^^^^^^

    SyntaxError: Unexpected token 'export'

    > 1 | import {cross, max, sum} from "d3-array";
        | ^
      2 | import {HistogramEntry} from "@/types/State";
      3 |
      4 | export type HitTransformer = (hitsplat: number) => HitDistribution;

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1505:14)
      at Object.<anonymous> (src/lib/HitDist.ts:1:1)
      at Object.<anonymous> (src/lib/CombatCalc.ts:3:1)
      at Object.<anonymous> (src/tests/utils/TestUtils.ts:4:1)
      at Object.<anonymous> (src/tests/CombatCalc.test.ts:3:1)`